tests/e2e/tests/pilot/authn_policy_test.go

```diff
@@ -22,9 +22,6 @@ import (
 
 func TestMTlsWithAuthNPolicy(t *testing.T) {
 	if tc.Kube.AuthEnabled {
-		// mTLS is now enabled via CRDs, so this test is no longer needed (and could cause trouble due
-		// to conflicts of policies)
-		// The whole authn test suites should be rewritten after PR #TBD for better consistency.
```

Comment:
Has this whole auth test been rewritten elsewhere? If so, then I'd give this one a 👍

Reply:
I think this file already cover all the authentication mTLS features, not sure what else we need to add, @diemtvu thoughts?